### DISCUSSION TITLE
How do I add an onscroll method?

### DISCUSSION THREAD

**Discussion by BraydenTW** [idiotWu/smooth-scrollbar#340] [Q&A] [answered]
How do I add an onscroll method?
## Environment
- Browser: chrome
- Version of smooth-scrollbar: 8.5.2

## Issue Summary
I can't figure out how to add an onScroll method to the scrollbar

**Comment by dolphin-wood** [comment, marked as answer]
Try [`scrollbar.addListener()`](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbaraddlistener).

**Comment by BraydenTW** [comment]
There is no scrollbar. So instead I checked when the position of the <div> changed when I scrolled and that worked fine. Can you make something like that a feature?
